tests/client.rs

```diff
@@ -28,3 +28,43 @@ async fn init_accounts_for_alice_and_bob() {
     assert_that!(got_alice_balance).is_equal_to(want_alice_balance);
     assert_that!(got_bob_balance).is_equal_to(want_bob_balance);
 }
+
+// This test should really be in wallet.rs but its easier to write if we have
+// access to the initialise Alice/Bob accounts.
+#[tokio::test]
+async fn transfer_and_check_tx_key() {
+    let docker = Cli::default();
+    let container = Container::new(&docker);
+    let cli = Client::new(container.monerod_rpc_port, container.wallet_rpc_port);
+
+    let balance_alice = 1000;
```

Comment:
The problem is here: Alice simply does not have enough money: 
You've added a field to `TransferParams` called `priority` and default to `0`. I'm not sure how monero-wallet-rpc comes up with a fee based on this, but sending a transaction resulted in a fee of `336_780_000`. Obviously, she can't pay for this. Hence increase the value to 1 monero and it will work: 

```suggestion
    let balance_alice = 1_000_000_000_000;
```

I had a hunch that this was the error and simply increased the value. It worked, so I reduced it again until I saw this error on the logs: 

```
2020-09-23 22:39:03.019	E needed_money + min_fee > balance_subtotal. THROW EXCEPTION: error::not_enough_money
2020-09-23 22:39:03.020	W /data/monero.git/src/wallet/wallet2.cpp:9838:N5tools5error16not_enough_moneyE: not enough money, available = 0.000100000000, tx_amount = 0.000010000000
```

Reply:
Nice work, I only increased it to 1_000_000 and never got this, well done. How did you get the fee amount (its not in the posted output above)?

Reply:
I saw it in the logs: 

```
transfer RPC response: {
  "id": "1",
  "jsonrpc": "2.0",
  "result": {
    "amount": 100,
    "fee": 336780000,
    "multisig_txset": "",
    "tx_blob": "",
    "tx_hash": "dd6a8c43e0edf8655cacb0e871bf2072112dd88391c4301ad582af8e8f53dcb6",
    "tx_key": "a9fb89f5191f9a5c59f025c9d19142fcabc1513bea48eaebf412557ce3649a05",
    "tx_metadata": "",
    "unsigned_txset": "",
    "weight": 1436
  }
}
```